src/awsTest/kotlin/com/ekino/oss/errors/RestExceptionHandlerTest.kt

```diff
@@ -24,9 +27,16 @@ class RestExceptionHandlerTest {
   fun should_get_no_such_key_exception() {
     mockMvc.perform(get("$RESOLVED_ERROR_PATH/no-such-key-exception"))
       .andExpect(status().isNotFound)
-      .andExpect(jsonPath("$.status", `is`(HttpStatus.NOT_FOUND.value())))
-      .andExpect(jsonPath("$.code", `is`("error.not_found")))
-      .andExpect(jsonPath("$.message", `is`(HttpStatus.NOT_FOUND.reasonPhrase)))
-      .andExpect(jsonPath("$.description", `is`("Message for developers")))
+      .andExpect(content().string(
+        jsonMatcher(loadJson())
```

Comment:
As the tests are in Kotlin, I don't think that's really interesting to extract the json into dedicated resource files.

You could remove every `loadJson` methods/utilities and directly use raw strings to be directly readable.
Example:
```suggestion
        jsonMatcher(
            """
            {
              "status": 404,
              "code": "error.not_found",
              "message": "Not Found",
              "description": "Message for developers",
              "errors": [],
              "globalErrors": [],
              "service": "myApp : GET /test/error/no-such-key-exception",
              "stacktrace": ""
            }
            """.trimIndent()
        )
```

What do you think of it?

Reply:
While json aren't too big I agree with @leomillon on this.

Reply:
I'm okay with you guys, since the json are not too large we can use raw strings. I will change that.